### DISCUSSION TITLE
Moving Stage Addons to Left

### DISCUSSION THREAD

**Discussion by bobby-sills** [ScratchAddons/ScratchAddons#5440] [Ideas]
<img width="507" alt="Screenshot 2022-12-11 at 14 21 57" src="https://user-images.githubusercontent.com/76852105/206891285-8edcda05-58b7-47b4-8f0c-365d7bf2622c.png">
As you can see in this photo, the menu addons above the stage are a bit crowded. This is not the greatest, as a lot of extensions use the top of the stage. One solution to fix this is to all, or some of the tools to the left, instead of having it all be in the stage.
<img width="674" alt="Screenshot 2022-12-11 at 14 28 14" src="https://user-images.githubusercontent.com/76852105/206891506-64d631d4-0dcf-4b4a-85a1-730899afa308.png">
This might be hard to make as it involves restructuring some of the divs, but I don't know. Please consider this addon

**Comment by DNin01** [comment]
Do you mean shifting the green flag and stop sign off to the side to give the addon information more space?

I think the green flag and stop sign should stay in the same spot, but there are probably better places for everything else, like to the left of the green flag or below the stage (#5155).

**Reply by WorldLanguages** [reply]
> Do you mean shifting the green flag and stop sign off to the side to give the addon information more space?

I have the same question. The screenshots are confusing.

**Reply by bobby-sills** [reply]
I didn't mean to shift the green flag, what I meant was that the addons on the stage should go to the left of the green flag. Because on my screen there's a lot a space over there

**Reply by Secret-chest** [reply]
Would that be practical? Please offer a setting to revert it, as I wouldn't get used to it.

**Reply by DNin01** [reply]
Maybe it should adjust automatically based on screen size.

**Reply by WorldLanguages** [reply]
This would only make sense for addons that don't show their UI while on the project page, since they wouldn't fit there either.
Addons that only show in the editor and/or fullscreen could, theoretically, use that space, if the window is wide enough.
